### DISCUSSION TITLE
Welcome to servicebus-csharp-function-dataverse Discussions!

### DISCUSSION THREAD

**Discussion by rpothin** [rpothin/servicebus-csharp-function-dataverse#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by klentgwapz** [comment]
https://github.com/klentgwapz/help_me_bisection_method/issues/1#issue-1410154094
Can you help me with this bisection method? I'm stuck at iteration 1 and can't find the another iterations. please help

**Reply by rpothin** [reply]
Hi @klentgwapz,

I am sorry, but I don't think there is an understandable link between your request and the work done in this repository.
But I wish you good luck with your problem.

> Note: I don't even know what the bisection method is.

Have a great day.
